public/app/features/alerting/unified/__snapshots__/RuleEditorCloudRules.test.tsx.snap

```diff
@@ -11,7 +11,7 @@ exports[`RuleEditor cloud can create a new cloud alert 1`] = `
       ],
     ],
     "method": "GET",
-    "url": "https://mimir.local:9000/api/v1/status/buildinfo",
+    "url": "http://localhost/api/datasources/proxy/uid/mimir/api/v1/status/buildinfo",
```

Comment:
Snapshots changed as I used the same mock datasource for Mimir. It isn't performing any different logic though, the previous datasource was pointing to the `mimir.local:9000` address, but that was causing MSW to not be able to intercept the handlers, so it's easier to just have it still point to localhost